packages/select/src/MenuButton/MenuButton.styles.ts

```diff
@@ -201,3 +212,29 @@ export const menuButtonErrorStyle: Record<Theme, string> = {
     }
   `,
 };
+
+export const validColor: Record<Theme, string> = {
+  [Theme.Light]: palette.green.dark1,
+  [Theme.Dark]: palette.green.base,
+};
+
+export const menuButtonValidStyle: Record<Theme, string> = {
+  [Theme.Light]: css`
+    border-color: ${errorColor[Theme.Light]};
```

Comment:
this feels verbose, no? we're already in theme.light i think you can just use our tokens here